### DISCUSSION TITLE
Library - Alphabet bar not working in landscape Mode

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#1776] [Fixed Issues]
The alphabet scrolling bar does not work in landscape mode on local screen or remote tablet.

**Comment by hjfisch** [comment]
That's right .
If you have double letters like AB , CD ,etc , it does not work in any mode .

**Comment by rern** [comment]
Please try the UPDATE.

**Comment by hjfisch** [comment]
Working now .

**Comment by R101G** [comment]
Yes, back to normal now.
